share/commands/agent-run

```diff
@@ -9,31 +9,103 @@
 # @man +
 # @man *-v*: run the agent in verbose mode, prints detailed information
 # @man +
+# @man *-d*: run the agent in debug mode, prints very detailed information
+# @man +
 # @man *-c*: run the agent without color output
+# @mab +
+# @man *-r*: run the agent with raw output
+# @mab +
+# @man *-m*: run the agent with multiline output
 # @man +
 # @man *-b*: run the agent on a specific bundle
+# @man +
+# @man *-D*: define a class for this run
 
 VERBOSITY=""
-COLOR="-C"
+COLOR="-Calways"
 BUNDLE=""
+CLASS=""
+MULTILINE=0
+SUMMARY_ONLY=0
+DEBUG_CLASS="-D debug"
+DISPLAY_INFO=0
+PRETTY_FILTER="/opt/rudder/share/lib/reports.awk"
+
+UUID=$(cat /opt/rudder/etc/uuid.hive 2>/dev/null)
+[ $? -ne 0 ] && UUID="Not yet configured"
+
+## Colors configuration (enable colors only if stdout is a terminal)
```

Comment:
We should probably have a library in rudder-agent for all the common stuff (like colors).